en/material-system/pass-parameter-list.md

```diff
@@ -154,6 +160,10 @@ Then the migration is guaranteed to execute, regardless of the existing data.<br
 
 > **Note**: Migration in force mode will execute in every database event, which is basically every mouse click in editor. So use it as a quick-and-dirty test measure, and be sure not to submit effect files with force mode migrations into version control.
 
+Again here are the bottomline rules about preventing potential data losses:<br>
+* Property removal will happen iff you specify the `removeImmediately` entry explicitly.
```

Comment:
iff
-->
if

Reply:
it's a shorthand for [if and only if](https://en.wikipedia.org/wiki/If_and_only_if), I thought that was common enough to be used...

Reply:
it's not common here unless you are a geek. I don't know if newbies understand this.

Reply:
iif
--> If, and only if,

Reply:
I disagree about the geek part but, alright, if that could make things better.

Reply:
I asked my wife and a few friends in group chat today if they knew ‘iff’ and nobody did.

Reply:
![197923DD-246C-4F6A-AD95-7E7F5DF6A090](https://user-images.githubusercontent.com/7491758/98619235-d363b100-22d0-11eb-897e-b07d72351f16.png)

Reply:
I know you are correct but it's too isoteric to be used

Reply:
Haha, I admit it's not an everyday term for everybody but it is actually pretty common in the academic area, no matter the field of study. So by instinct I found 'geek' kind of over the top I guess.

Reply:
But anyways the document is not targeted at professors or researchers, so the point here is layman's term should always be preferred.